tests/shutdown/cleanup_before_shutdown.pm

```diff
@@ -25,7 +25,7 @@ sub run {
     # Please see https://freedesktop.org/wiki/Software/systemd/Debugging/#index2h1 for the details.
     # Boot options that are required to make logs more detalized are located in 'bootloader_setup.pm'
     if (get_var('DEBUG_SHUTDOWN')) {
-        assert_script_run "echo -e '#!/bin/sh\\ndmesg > /dev/$serialdev' > /usr/lib/systemd/system-shutdown/debug.sh";
+        assert_script_run "echo -e '#!/bin/sh\\necho --- dmesg log ---  > /dev/$serialdev\\ndmesg  >> /dev/$serialdev\\necho --- journactl log ---  >> /dev/$serialdev\\njournalctl >> /dev/$serialdev' > /usr/lib/systemd/system-shutdown/debug.sh";
```

Comment:
Idea looks good, the only thing I don't like this enormous line here. What's about extracting like:
```perl
    my $script = <<"END_SCRIPT";
    echo -e '#!/bin/sh\\necho --- dmesg log ---  > /dev/$serialdev\\ndmesg  >> /dev/$serialdev\\n
    echo --- journactl log ---  >> /dev/$serialdev\\njournalctl >> /dev/$serialdev'
 > /usr/lib/systemd/system-shutdown/debug.sh    
END_SCRIPT

assert_script_run $script;
```

Reply:
thanks